pl_bolts/models/detection/faster_rcnn/faster_rcnn_module.py

```diff
@@ -46,6 +49,8 @@ def __init__(
         self,
         learning_rate: float = 0.0001,
         num_classes: int = 91,
+        backbone: str = None,
```

Comment:
mind adding `from typing import Optional` and
```suggestion
        backbone: Optional[str] = None,
```
https://github.com/PyTorchLightning/pytorch-lightning/pull/4164

Reply:
Yes definitely will add these :)